src/EntityGraphQueries.ts

```diff
@@ -19,6 +19,133 @@ const EntityGraphQueries: Record<string, string> = {
   + '    uts\n'
   + '  }\n'
   + '}',
+
+  stakeIntentDeclareds: 'query ($contractAddress: String!, $uts: BigInt!, $skip: BigInt!, $limit:'
+  + ' BigInt!) {\n'
+  + 'stakeIntentDeclareds(skip: $skip, first:$limit, orderDirection: asc, where: {contractAddress:'
+  + ' $contractAddress, uts_gt: $uts}, '
+  + '    orderDirection: asc,'
+  + '    limit: 100) {\n'
+  + '    id\n'
+  + '    amount\n'
```

Comment:
If we have a select * kinda thing in the query it would help us here.

Reply:
```One of the main benefits of using GraphQL is actually to prevent overfetching.```
https://stackoverflow.com/questions/34199982/how-to-query-all-the-graphql-type-fields-without-writing-a-long-query